features/core-download.feature

```diff
@@ -295,3 +295,36 @@ Feature: Download WordPress
     """
     /root-level-directory/
     """
+
+  Scenario: Core download without the wp-content dir
+    Given an empty directory
+    And an empty cache
+
+    When I run `wp core download --skip-content`
+    And save STDOUT 'Downloading WordPress ([\d\.]+)' as {VERSION}
+    Then the wp-settings.php file should exist
+    And the {SUITE_CACHE_DIR}/core/wordpress-{WP_VERSION-latest}-without-content-en_US.zip file should exist
+
+
+  Scenario: Core download without the wp-content dir should error for non US locale
+    Given an empty directory
+    And an empty cache
+
+    When I run `wp core download --skip-content --locale=nl_NL`
+    Then STDOUT should contain:
```

Comment:
This should be `Then STDERR should contain`

Reply:
Thanks, I've just fixed.